### ISSUE TITLE
Allow users to set multiple frequency plans only in the same band

### ISSUE THREAD

**Issue by KrishnaIyer** [TheThingsNetwork/lorawan-stack#6644]
### Summary

This is an extension of https://github.com/TheThingsNetwork/lorawan-stack/pull/6615

### Current Situation

Users can mix and match frequency plans from different bands.

### Why do we need this? Who uses it, and when?

When users set multiple FPs from different bands, the backend will return an error.
Since the console already fetches the Band ID, it would be easier to validate this client side.

### Proposed Implementation

- Check that every new plan added after the first one has the same band ID as the first one. 
- Bonus points if the dropdown is only populated with frequency plans in the same band as the first one.

### Contributing

- [ ] I can help by doing more research.
- [ ] I can help by implementing the feature after the proposal above is approved.
- [X] I can help by testing the feature before it's released.

### Code of Conduct

- [X] I agree to follow TTN's [Community Code of Conduct](https://github.com/TheThingsNetwork/Manifesto/blob/master/CodeOfConduct.md).

**Comment by KrishnaIyer** [comment]
This is blocked on #6686

**Comment by KrishnaIyer** [comment]
@ryaplots: Please see the backend PR #6709.
